src/main/java/Duke.java

```diff
@@ -1,10 +1,36 @@
+import java.util.Scanner;
+//import java.util.ArrayList;
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke\n" + "What can I do for you?");
```

Comment:
Is it better to extract the whole things out and make it as a showWelcomeMessage() method?